tests/framework/Optimizers/GeneticAlgorithms/gold/discrete/unconstrained/MaxwoReplacementTwoPointsCrossover/opt_export_0.csv

```diff
@@ -0,0 +1,402 @@
+1.0,5.0,3.0,6.0,2.0,4.0,78.0,2.0,18.0,156.0,17.0,accepted,0.0
+1.0,6.0,4.0,3.0,5.0,2.0,74.0,2.0,18.0,148.0,17.0,accepted,0.0
+1.0,5.0,3.0,6.0,4.0,2.0,76.0,2.0,18.0,152.0,17.0,accepted,0.0
+1.0,5.0,4.0,6.0,2.0,3.0,75.0,2.0,18.0,150.0,17.0,accepted,0.0
+1.0,6.0,3.0,4.0,5.0,2.0,75.0,2.0,18.0,150.0,17.0,accepted,0.0
+1.0,5.0,3.0,6.0,4.0,2.0,76.0,2.0,18.0,152.0,17.0,accepted,0.0
+1.0,2.0,4.0,3.0,6.0,5.0,89.0,3.0,19.0,178.0,18.0,accepted,0.0
+3.0,2.0,4.0,1.0,5.0,6.0,84.0,3.0,19.0,168.0,18.0,accepted,0.0
+4.0,2.0,3.0,1.0,5.0,6.0,82.0,3.0,19.0,164.0,18.0,accepted,0.0
+2.0,1.0,4.0,3.0,6.0,5.0,88.0,3.0,19.0,176.0,18.0,accepted,0.0
+1.0,2.0,4.0,3.0,6.0,5.0,89.0,3.0,19.0,178.0,18.0,accepted,0.0
+1.0,2.0,4.0,3.0,6.0,5.0,89.0,3.0,19.0,178.0,18.0,accepted,0.0
+1.0,2.0,4.0,3.0,6.0,5.0,89.0,3.0,19.0,178.0,18.0,accepted,0.0
+3.0,2.0,4.0,1.0,5.0,6.0,84.0,3.0,19.0,168.0,18.0,accepted,0.0
+4.0,2.0,3.0,1.0,5.0,6.0,82.0,3.0,19.0,164.0,18.0,accepted,0.0
+2.0,3.0,4.0,5.0,6.0,1.0,76.0,3.0,19.0,152.0,18.0,accepted,0.0
+3.0,2.0,4.0,1.0,5.0,6.0,84.0,3.0,19.0,168.0,18.0,accepted,0.0
+1.0,2.0,4.0,3.0,6.0,5.0,89.0,3.0,19.0,178.0,18.0,accepted,0.0
+3.0,2.0,4.0,1.0,5.0,6.0,84.0,3.0,19.0,168.0,18.0,accepted,0.0
+3.0,2.0,4.0,1.0,5.0,6.0,84.0,3.0,19.0,168.0,18.0,accepted,0.0
+2.0,1.0,4.0,3.0,6.0,5.0,88.0,3.0,19.0,176.0,18.0,accepted,0.0
+1.0,4.0,3.0,5.0,6.0,2.0,80.0,3.0,19.0,160.0,18.0,accepted,0.0
+2.0,3.0,4.0,5.0,6.0,1.0,76.0,3.0,19.0,152.0,18.0,accepted,0.0
+4.0,2.0,3.0,1.0,5.0,6.0,82.0,3.0,19.0,164.0,18.0,accepted,0.0
+3.0,2.0,4.0,1.0,6.0,5.0,83.0,3.0,19.0,166.0,18.0,accepted,0.0
+3.0,2.0,4.0,1.0,5.0,6.0,84.0,3.0,19.0,168.0,18.0,accepted,0.0
+4.0,2.0,1.0,3.0,5.0,6.0,84.0,9.0,20.0,168.0,19.0,accepted,0.0
+4.0,2.0,1.0,3.0,6.0,5.0,83.0,9.0,20.0,166.0,19.0,accepted,0.0
+4.0,2.0,1.0,3.0,6.0,5.0,83.0,9.0,20.0,166.0,19.0,accepted,0.0
+4.0,2.0,1.0,3.0,5.0,6.0,84.0,9.0,20.0,168.0,19.0,accepted,0.0
+3.0,1.0,6.0,5.0,4.0,2.0,75.0,9.0,20.0,150.0,19.0,accepted,0.0
+4.0,2.0,6.0,1.0,5.0,3.0,73.0,9.0,20.0,146.0,19.0,accepted,0.0
+4.0,2.0,1.0,3.0,6.0,5.0,83.0,9.0,20.0,166.0,19.0,accepted,0.0
+4.0,2.0,1.0,3.0,5.0,6.0,84.0,9.0,20.0,168.0,19.0,accepted,0.0
+4.0,2.0,1.0,3.0,6.0,5.0,83.0,9.0,20.0,166.0,19.0,accepted,0.0
+4.0,2.0,1.0,3.0,6.0,5.0,83.0,9.0,20.0,166.0,19.0,accepted,0.0
+3.0,1.0,5.0,4.0,2.0,6.0,82.0,9.0,20.0,164.0,19.0,accepted,0.0
+2.0,4.0,1.0,3.0,6.0,5.0,85.0,9.0,20.0,170.0,19.0,accepted,0.0
+4.0,2.0,1.0,3.0,6.0,5.0,83.0,9.0,20.0,166.0,19.0,accepted,0.0
+4.0,2.0,1.0,3.0,6.0,5.0,83.0,9.0,20.0,166.0,19.0,accepted,0.0
+3.0,2.0,1.0,4.0,5.0,6.0,87.0,9.0,20.0,174.0,19.0,accepted,0.0
+4.0,1.0,2.0,3.0,6.0,5.0,84.0,9.0,20.0,168.0,19.0,accepted,0.0
+4.0,2.0,1.0,3.0,6.0,5.0,83.0,9.0,20.0,166.0,19.0,accepted,0.0
+4.0,2.0,1.0,3.0,6.0,5.0,83.0,9.0,20.0,166.0,19.0,accepted,0.0
+4.0,2.0,3.0,5.0,6.0,1.0,73.0,9.0,20.0,146.0,19.0,accepted,0.0
+3.0,1.0,2.0,4.0,5.0,6.0,88.0,9.0,20.0,176.0,19.0,accepted,0.0
+1.0,2.0,3.0,4.0,5.0,6.0,91.0,10.0,20.0,182.0,19.0,final,0.0
```

Comment:
couple of questions here:
1. why the con_objective is not changing?
2. Will this test converge to the analytical solution? I saw you have marked the test as analytic.

Reply:
1. Conv_objective is a boolean that is tro when objective value converges to a user given value which is not the case here that's why it is false. (I think it should be False not zero, but this is the case in most optimizers.)
2. The analytic solution is 1,2,3,4,5,6 and objective is 91, which is exactly the solution.

Reply:
I see in the test the target objective value is '-1', does it mean when the objective value converge to '-1', the con_objective will become True? Another question is, in the test, the type of optimization is assigned to 'max' (see below), but why the objective value is '-1'. @Jimmy-INL  

![image](https://user-images.githubusercontent.com/7321071/145058645-a8085839-940f-4ec0-9869-ad3d8f2e3345.png)

Reply:
When I know that the objective is a positive value and I select conv_obj = -1, this is just to force the algorithm to continue till the iteration limit is reached. I can remove this from this test if it causes confusion.